inline-caching/README.md

```diff
@@ -0,0 +1,247 @@
+# Inline Cache with GemFire
```

Comment:
If postgres is mentioned outside of running commands, it should capitalized -- Postgres

Reply:
capitalized